Sources/AppCoinsSDK/Models/Services/Attribution/AttributionClient.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AttributionClient.swift
+//
+//
+//  Created by Graciano Caldeira on 12/07/2024.
+//
+
+import Foundation
+
+internal class AttributionClient: AttributionService {
+    
+    private let endpoint: String
+    
+    internal init(endpoint: String = BuildConfiguration.attributionServiceBaseURL) {
+        self.endpoint = endpoint
+    }
+    
+    internal func getAttribution(bundleID: String, result: @escaping (Result<AttributionRaw, AttributionServiceErrors>) -> Void) {
+        if #available(iOS 16.0, *) {
```

Comment:
Let's not create technical debt by limiting calls to iOS versions. Simply build the query URL manually.